.vscode/launch.json

```diff
@@ -115,7 +116,8 @@
         "npm_and_yarn",
         "nuget",
         "python",
-        "terraform"
+        "terraform",
+        "github_actions"
```

Comment:
This makes perfect sense 😄 What do you think of putting this ecosystem in its place alphabetically? I think it'd be a nice touch and would slim the diff down 🤏

Reply:
Let me do that!